modules/virt-enabling-preallocation-globally.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc
+
+[id="virt-enabling-preallocation-globally_{context}"]
+= Enabling preallocation globally
+
+You can enable preallocation mode for the Containerized Data Importer (CDI) for the whole cluster by adding the `prellocation` field to the `CDI` object.
+
+.Prerequisites
+
+* Install the OpenShift client (`oc`).
+
+.Procedure
+
+. Use the `oc` client to edit the `CDI` object:
++
+[source,terminal]
+----
+$ oc edit cdi
+----
+
+. Edit the `spec.config.preallocation` field so that it has a value of `true`:
```

Comment:
Change:
Edit the `spec.config.preallocation` field so that it has a value of `true`:
To:
Edit the `spec.config.preallocation` field and set it with value of `true`: